cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SvInjectionConversion.java

```diff
@@ -49,6 +50,7 @@ public void convert() {
         identify(adder);
         connect(adder);
         Load load = adder.add();
+        load.setProperty(Conversion.PROPERTY_IS_CREATED_FOR_SV_INJECTION, "true");
```

Comment:
I don't think you need to do that for fictitious loads (and in fact, I think we shouldn't as it makes the model heavier): it is not a problem in my opinion to export fictitious loads as SV injections in every case as we do not lose any IDs. The issue with switches was the possibility to lose IDs after export/import (for example).

Reply:
The aim of adding the property was to be sure that we do not ignore in the export other fictitious loads that may have been created in the model for other purposes.
Native IIDM cases could contain loads marked as "fictitious" for some reason by the users, but they want them to be exported to the EQ as regular loads. 
If we remove this property we won't we able to distinguish between these two situations, and we will ignore all fictitious loads in the export.
Is it ok for you?

Reply:
For me, it is not an issue as no information loss is performed in this case. @annetill do you think it is of importance?

Reply:
I've just discussed with Anne: we agree to export fictitious loads as SvInjection in any case, at least for now (we will see later if we have a new use case)

Reply:
Ok. I will remove the property for fictitious loads. All fictitious loads will be exported as `SvInjection`, they will not show in the EQ file.